gradle.properties

```
@@ -1,2 +1,28 @@
 kotlin.code.style=official
 org.gradle.jvmargs=-Xmx2G
+# Versions
+bouncycastleVersion=1.68
+commonsCodecVersion=1.15
+commonsCompressVersion=1.20
+gradleWrapperVersion=6.8.1
+hamkrestJsonVersion=1.1.2
```

I don't think this is needed any more...

```suggestion
commonsCompressVersion=1.20
hamkrestJsonVersion=1.1.2
```

---

Here ya go:
```
🌼 ./gradlew wrapper

> Configure project :
Reckoned version: 0.66.0-dev.0.12+f3fc92f

BUILD SUCCESSFUL in 1s
1 actionable task: 1 up-to-date

boxley@nurnen ~/src/batect
🌼 grep gradle gradle.properties 
org.gradle.jvmargs=-Xmx2G
```